R/network.R

```diff
@@ -296,6 +296,7 @@ nearest_node <- function(network, target) {
 #' @param target The target geometry
 #'
 #' @return A spatial network object
+#' @export
```

Comment:
I think this is a utility function for CRiSp, not sure whether we should make it public as it is a thin wrapper of sfnetworks. I see that you have to do it, because we are using it in  the segmentation vignette. A better approach to me would be to keep this private, and, in the vignette to do the filtering on the edges (streets + railways) before building the network. But it's a rather minor point and we can maybe address it later - let's leave this as is and add an issue?